src/components/QuickOpenModal.js

```diff
@@ -353,9 +351,11 @@ export class QuickOpenModal extends Component<Props, State> {
       return null;
     }
 
-    let highlightedResults = results;
+    let highlightedResults;
     if (results) {
       highlightedResults = this.fuzzyhighlight(query, results);
```

Comment:
How about `const highlightedResults = results ? this.fuzzyhighlight(query, results) : []; `